frontend/src/components/visual-editor/form/inputs/message/QuickRepliesInput.tsx

```diff
@@ -47,16 +48,9 @@ const QuickRepliesInput: FC<QuickRepliesInput> = ({
     const updatedQuickReplies = [...quickReplies];
 
     updatedQuickReplies.splice(index, 1);
+    // Set updated state only if it's greater than minInput
```

Comment:
Great Job! Just remove this small comment and we are good to go.
Thank you for your contribution!